src/ContainerLogs.jsx

```diff
@@ -165,8 +166,21 @@ class ContainerLogs extends React.Component {
 
     render() {
         let element = <div className="container-logs" ref={this.logRef} />;
-        if (this.state.errorMessage)
+        const { systemd_unit, uid } = this.props;
+
+        if (this.state.errorMessage) {
             element = <EmptyStatePanel icon={ExclamationCircleIcon} title={this.state.errorMessage} />;
+        } else if (systemd_unit !== undefined && uid === 0) {
+            element = (
+                <>
+                    {element}
+                    <Button variant="link" isInline className="pf-v6-u-mt-sm" onClick={
+                        () => cockpit.jump(`/system/logs/#/?priority=info&_SYSTEMD_UNIT=${systemd_unit}`)}>
+                        {cockpit.format(_("View $0 logs"), systemd_unit)}
+                    </Button>
```

Comment:
Can we make this a link with a `href` instead? It's not great when buttons do navigations when there is no indication of it, and we can change the PF Button to be a link with `href`

Reply:
Ack. Fine for now then. It is something I'd love for us to fix in the future though